test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/TestAutoCreateSharedArchiveUpgrade.java

```diff
@@ -90,7 +90,7 @@ public static void main(String[] args) throws Throwable {
             try {
                 setupJVMs(Integer.parseInt(versions[i]));
                 doTest();
-            } catch (NumberFormatException e) {
+            } catch (RuntimeException e) {
                 throw new RuntimeException("Invalid AutoCreateSharedArchive JDK version: " + versions[i]);
```

Comment:
If catching a general exception here you should now add that as the cause of the new one.

Reply:
But why did you change this anyway?

Reply:
I figured I needed to catch the exception thrown by `setupJVMs()` but that isn't necessary. I will revert this.